src/vm.nit

```diff
@@ -46,6 +46,16 @@ class VirtualMachine super NaiveInterpreter
 	# The unique instance of the `MInit` value
 	var initialization_value: Instance is noinit
 
+	# The current frame used to store local variables of the current method executed
+	fun current_frame: NumberedFrame do return numbered_frames.first
+
+	# Is a return executed?
+	# Set this mark to skip the evaluation until the end of the specified method frame
```

Comment:
Reading this I'm not sure to understand the comment.

Am I supposed to set this myself? But the property is not `is writable`?